cdap-master/src/main/java/co/cask/cdap/master/startup/HBaseCheck.java

```diff
@@ -104,7 +107,7 @@ public void run() {
           throw new RuntimeException("Unable to determines cdap privileges as global admin in HBase.");
         }
       }
-      LOG.info("Hbase authorized is disabled. Skipping global admin check.");
     }
+    LOG.info("Hbase authorization is disabled. Skipping global admin check.");
```

Comment:
ok, so the reason we need the global admin check is that Pruning is on, and HBase authorization is on, too. To make this understandable to the user, that's what you want to log, above when you determine that you want to check, right now you only say pruning is on. And down here you only say HBase auth is on. Both of these are incomplete and should instead contain both pieces of information.